fvm/evm/stdlib/contract_test.go

```diff
@@ -283,6 +285,12 @@ func TestEVMEncodeABI(t *testing.T) {
 		OnDecodeArgument: func(b []byte, t cadence.Type) (cadence.Value, error) {
 			return json.Decode(nil, b)
 		},
+		OnMeterComputation: func(compKind common.ComputationKind, intensity uint) error {
+			if compKind == environment.ComputationKindEVMEncodeABI {
+				computation = intensity
```

Comment:
```suggestion
				computation += intensity
```

Reply:
Updated in https://github.com/onflow/flow-go/pull/5146/commits/32e6b6f0b533f8de085671e5fff80c4e4587118f